snmp.yml

```diff
@@ -2606,147 +3270,206 @@ ddwrt:
+      1.3.6.1.4.1.2021.11.8
   - name: ssCpuUser
     oid: 1.3.6.1.4.1.2021.11.9
     type: gauge
+    help: The percentage of CPU time spent processing user-level code, calculated
+      over the last minute - 1.3.6.1.4.1.2021.11.9
   - name: ssCpuSystem
     oid: 1.3.6.1.4.1.2021.11.10
     type: gauge
+    help: The percentage of CPU time spent processing system-level code, calculated
+      over the last minute - 1.3.6.1.4.1.2021.11.10
   - name: ssCpuIdle
     oid: 1.3.6.1.4.1.2021.11.11
     type: gauge
+    help: The percentage of processor time spent idle, calculated over the last minute
+      - 1.3.6.1.4.1.2021.11.11
   - name: ssCpuRawUser
     oid: 1.3.6.1.4.1.2021.11.50
     type: counter
+    help: The number of 'ticks' (typically 1/100s) spent processing user-level code
+      - 1.3.6.1.4.1.2021.11.50
   - name: ssCpuRawNice
     oid: 1.3.6.1.4.1.2021.11.51
     type: counter
+    help: The number of 'ticks' (typically 1/100s) spent processing reduced-priority
+      code - 1.3.6.1.4.1.2021.11.51
   - name: ssCpuRawSystem
     oid: 1.3.6.1.4.1.2021.11.52
     type: counter
+    help: The number of 'ticks' (typically 1/100s) spent processing system-level code
+      - 1.3.6.1.4.1.2021.11.52
   - name: ssCpuRawIdle
     oid: 1.3.6.1.4.1.2021.11.53
     type: counter
+    help: The number of 'ticks' (typically 1/100s) spent idle - 1.3.6.1.4.1.2021.11.53
   - name: ssCpuRawWait
     oid: 1.3.6.1.4.1.2021.11.54
     type: counter
+    help: The number of 'ticks' (typically 1/100s) spent waiting for IO - 1.3.6.1.4.1.2021.11.54
   - name: ssCpuRawKernel
     oid: 1.3.6.1.4.1.2021.11.55
     type: counter
+    help: The number of 'ticks' (typically 1/100s) spent processing kernel-level code
+      - 1.3.6.1.4.1.2021.11.55
   - name: ssCpuRawInterrupt
     oid: 1.3.6.1.4.1.2021.11.56
     type: counter
+    help: The number of 'ticks' (typically 1/100s) spent processing hardware interrupts
+      - 1.3.6.1.4.1.2021.11.56
   - name: ssIORawSent
     oid: 1.3.6.1.4.1.2021.11.57
     type: counter
+    help: Number of blocks sent to a block device - 1.3.6.1.4.1.2021.11.57
   - name: ssIORawReceived
     oid: 1.3.6.1.4.1.2021.11.58
     type: counter
+    help: Number of blocks received from a block device - 1.3.6.1.4.1.2021.11.58
   - name: ssRawInterrupts
     oid: 1.3.6.1.4.1.2021.11.59
     type: counter
+    help: Number of interrupts processed - 1.3.6.1.4.1.2021.11.59
   - name: ssRawContexts
     oid: 1.3.6.1.4.1.2021.11.60
     type: counter
+    help: Number of context switches - 1.3.6.1.4.1.2021.11.60
   - name: ssCpuRawSoftIRQ
     oid: 1.3.6.1.4.1.2021.11.61
     type: counter
+    help: The number of 'ticks' (typically 1/100s) spent processing software interrupts
+      - 1.3.6.1.4.1.2021.11.61
   - name: ssRawSwapIn
     oid: 1.3.6.1.4.1.2021.11.62
     type: counter
+    help: Number of blocks swapped in - 1.3.6.1.4.1.2021.11.62
   - name: ssRawSwapOut
     oid: 1.3.6.1.4.1.2021.11.63
     type: counter
-  - name: ssCpuRawSteal
```

Comment:
I'm generating the yaml using the latest mibs as per the [generator readme]( http://www.net-snmp.org/docs/mibs/UCD-SNMP-MIB.txt) which has these metrics listed yet they are removed for some reason.